Simcoe/Adobe/Adobe.swift

```diff
@@ -47,20 +47,36 @@ extension Adobe: EventTracking {
 
 extension Adobe: LifetimeValueIncreasing {
 
-    /// Increases the lifetime value of the key by the specified amount.
+    /// Increments the property.
     ///
     /// - Parameters:
-    ///   - amount: The amount to increase that lifetime value for.
-    ///   - item: The optional item to extend.
-    ///   - properties: The optional additional properties.
+    ///   - property: The property.
+    ///   - value: The amonut to increment the property by.
     /// - Returns: A tracking result.
-    public func increaseLifetimeValue(byAmount amount: Double, forItem item: String?, withAdditionalProperties properties: Properties?) -> TrackingResult {
+    public func increment(property: String?, value: Double, withAdditionalProperties properties: Properties?) -> TrackingResult {
         var data = properties ?? Properties()
-        if let item = item {
-            data[item] = ""
+
+        if let property = property {
+            data[property] = ""
         }
 
-        ADBMobile.trackLifetimeValueIncrease(1, data: data)
+        ADBMobile.trackLifetimeValueIncrease(NSDecimalNumber(value: value), data: data)
+        return .success
+    }
+
+    /// Increments the properties.
+    /// All values must be number objects.
+    ///
+    /// - Parameter properties: The properties.
+    /// - Returns: A tracking result.
+    public func increment(properties: Properties, withAdditionalProperties data: Properties?) -> TrackingResult {
```

Comment:
It's kinda odd that the argument for `withAdditionalProperties` is called `data` here and `properties` in the previous function.